MarpSlides.md

```diff
@@ -0,0 +1,79 @@
+# Marp Preview
+
+This library provides a way to preview your Marp slides in a panel while you are editing your space.
+
+With Marp Preview, you can:
+- Preview your slides without leaving the context of your space
+- See how your slides look in real-time as you modify your markdown
+- Use the Marp Preview panel to navigate through your slides and see them in action
+
+See [source](https://community.silverbullet.md/t/marp-preview-plugin/741)
+
+```space-lua
+local LOG_ENABLE = false
+
+local function debug_log(message)
+  if LOG_ENABLE then
+    js.log("[DEBUG] " .. message)
+  end
+end
+
+local is_panel_visible = false
```

Comment:
Variable and function naming consistency: I went with camelCase style naming, so let's keep this consistency, so no snake_casing please.